guide/breaking-changes.md

```diff
@@ -40,6 +40,18 @@ From version `v0.3.3`, `@vite-pwa/nuxt` configures `dontCacheBustURLsMatching` i
 
 From version `v0.2.0`, `@vite-pwa/astro` configures `dontCacheBustURLsMatching` in a similar way to how `vite-plugin-pwa` does, but using the Astro's [build.assets](https://docs.astro.build/en/reference/configuration-reference/#buildassets) option (defaults to `_astro`).
 
+## @vite-pwa/sveltekit
+
+From version `v0.3.0`, `@vite-pwa/sveltekit` supports SvelteKit 2 (should also support SvelteKit 1).
```

Comment:
this isn't really a breaking change since it only adds functionality and doesn't remove any

Reply:
I need to change section title

Reply:
renamed breaking changes to change log